armi/reactor/tests/test_components.py

```diff
@@ -15,12 +15,16 @@
 """Tests functionalities of components within ARMI."""
 import copy
 import math
+import numpy as np
+from numpy.testing import assert_equal
 import unittest
+from unittest.mock import Mock
```

Comment:
Some intersparse third party imports w/ the standard library imports.

I think ARMI tries to follow the import pattern of
1. Standard library (unittest)
2. Third party (np)
3. ARMI

all separated by new lines.

If that's indeed the rule we're trying to follow, please move the numpy imports between standard library and armi

Reply:
https://github.com/terrapower/armi/pull/1990/commits/937057cb664ffc971bf525f0062156bc8fa30e05